src/citrine/resources/material_spec.py

```diff
@@ -87,3 +87,22 @@ class MaterialSpecCollection(DataConceptsCollection[MaterialSpec]):
     def get_type(cls) -> Type[MaterialSpec]:
         """Return the resource type in the collection."""
         return MaterialSpec
+
+    def get_runs(self, scope: str, id: str, per_page: int = 20) -> Iterator[dict]:
```

Comment:
Not a requirement, but I'd suggest the input parameters be similar to the `get` method in data_concepts:
So `scope="id"` by default.